package.json

```diff
@@ -9,7 +9,9 @@
   "bugs": {
     "url": "https://github.com/kazupon/vue-i18n/issues"
   },
-  "dependencies": {},
+  "dependencies": {
+    "vue-template-compiler": "^2.5.16"
```

Comment:
ah, sorry, 🙇 
Can you change the `devDependencies`?

Reply:
I did the modification and fixed up the commit on the dependency commit 👍